src/catalogue/category/catalogueCategoryTableView.component.tsx

```diff
@@ -122,6 +126,35 @@ const CatalogueCategoryTableView = (props: CatalogueCategoryTableViewProps) => {
       shape: 'rounded',
       variant: 'outlined',
     },
+
+    //Functions
+    renderCreateRowDialogContent: ({ table, row }) => {
+      return (
+        <>
+          <CatalogueCategoryDialog
+            open={true}
+            onClose={() => table.setCreatingRow(null)}
+            parentId={catalogueCategoryParentId ?? null}
+            type="add"
+            resetSelectedCatalogueCategory={() => table.setCreatingRow(null)}
+          />
```

Comment:
I think these coverage warnings can be resolved if they were tested in the unit tests. There is another for the systems table view.